fits_schema/fits_standard.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from .header import HeaderSchema, HeaderCard
+
+
+class FITSStandardHeaders(HeaderSchema):
```

Comment:
Since you implemented this group feature now, it might make sense to apply this here, e.g. have separate schema instances for `Base` / `Time` / `InstrumentMeta` etc.

Reply:
For now I think I'll keep these simple, since they will just be for schema validation.  In the vodf-schema package, I will fully define the FITS standard headers that we will use, in groups and with all metadata.  I could do all that here, but then it's harder to re-use them as they are all set to "optional".